src/Workspaces/Core/Portable/Remote/RemoteSupportedLanguages.cs

```diff
@@ -2,16 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
+using System.Diagnostics.CodeAnalysis;
 
-namespace Microsoft.CodeAnalysis.Remote
+namespace Microsoft.CodeAnalysis.Remote;
+
+internal static class RemoteSupportedLanguages
 {
-    internal static class RemoteSupportedLanguages
-    {
-        public static bool IsSupported(this string language)
-        {
-            return language is LanguageNames.CSharp or
-                   LanguageNames.VisualBasic;
-        }
-    }
+    public static bool IsSupported([NotNullWhen(true)] this string? language)
+        => language is LanguageNames.CSharp or LanguageNames.VisualBasic;
```

Comment:
I never like these null-accepting things like this. :cry: (Not blocking but I can still be sad in my own corner.)